### PR TITLE
Arrange task and question randomization

### PR DESCRIPTION
**Pull request by pantelicn**

### CODE REVIEW

File: src/app/shared/helpers/arrays.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function getRandom(min, max): number {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min) + min);
+}
+
+export function shuffleArray<Type>(arr: Array<Type>): Array<Type>  {
```
Comment (Luburic) [inline src/app/shared/helpers/arrays.ts:7]: Is this any particular algorithm?

It looks like knuth shuffle, but there are some differences in the code https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array although I'm not sure if they are significant.

**Review by Luburic — APPROVED** [review]
One minor comment to examine, but otherwise looks good.

File: src/app/shared/helpers/arrays.ts (hunk 1 above)
Comment (pantelicn) [reply src/app/shared/helpers/arrays.ts:7]: It is knuth shuffle. Do you want me to specify it in the comment above the function?

Comment (Luburic) [reply src/app/shared/helpers/arrays.ts:7]: No need, you're good to go :)

**Pull request merged**